ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/WrappingRule.kt

```diff
@@ -210,7 +210,11 @@ public class WrappingRule : Rule(
             // put entries on separate lines
             // TODO: group emit()s below with the one above into one (similar to ParameterListWrappingRule)
             for (c in node.children()) {
-                if (c.elementType == COMMA && !c.treeNext.isWhiteSpaceWithNewline()) {
+                if (
+                    c.elementType == COMMA &&
+                    !c.treeNext.isWhiteSpaceWithNewline() &&
+                    !(c.nextLeaf() is PsiWhiteSpace && c.nextLeaf()?.nextLeaf()?.isPartOfComment() == true)
```

Comment:
The test that you have added also succeeds without this specific code change. So either this code is not required or the test case does not adequately test the code.

Reply:
@paul-dingemans how are you determining that? When I delete this new condition the test fails
`Expecting empty but was:<[LintError(line=4, col=14, ruleId=wrapping, detail=Missing newline after ",")]>`

Reply:
Sorry, my comment was not entirely clear. I think that I meant that `c.nextLeaf() is PsiWhiteSpace && ` was redundant. After playimg around a bit more, I realized that your solution assumed the presence of a whitespace (without newline) before the comment. I have pushed a changed for this to your branch.